charts/connect/CHANGELOG.md

```diff
@@ -12,6 +12,19 @@
 
 ---
 
+[//]: # (START/LATEST)
+# v1.10.0
+
+## Features
+* Add replicas definition to the Connect Deployment {#121}
```

Comment:
I think we can merge this one with the fix, since it's all about providing this functionality properly.

Here's my suggestion:
```suggestion
* Add replicas definition to the Connect Deployment {#121, #128}
```